### ISSUE TITLE
SWARM-REGISTER: stale takeover PDR attributable-subset source admission

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#2007]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
ROLE: AUTONOMOUS SENIOR/PRINCIPAL IMPLEMENTER
STATUS: REGISTERED_SELECTED_PRECLAIM_PENDING_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
MERGE_AUTHORITY: false
TRAINING_CLAIM: false
PAID_COMPUTE_AUTHORITY: false
PRODUCT_MUTATION: false until postclaim arbitration WIN

SELECTED_SEMANTIC:
D03|COMMON-PILE-PUBLIC-DOMAIN-REVIEW|ATTRIBUTABLE-SUBSET-SOURCE-ADMISSION|V1
STALE_PREDECESSOR: #1757
PREDECESSOR_LAST_UPDATE_UTC: 2026-09-14T00:08:07Z
PREDECESSOR_DEFAULT_LEASE_EXPIRED_UTC: 2026-09-14T06:08:07Z
LIVE_MAIN_AT_REGISTRATION: fc57531f7f8e7ef03323df641d8196af6c573c8b

SHORT LIVE AUDIT / CANDIDATE COMPARISON:
1. Physical Nomis1864/Sonnet-free clean successor rebuild is the global P0 but is actively Product-owned by #1951; YIELD.
2. Central learned-20M integration is actively owned by #1955/lawful successor; YIELD.
3. Multiple nearest released launch/data heads (PR463, PR476, PR1787, PR1806, PR1851, Windows carrier) have live Product/audit/prequalification owners; YIELD rather than duplicate.
4. PDR attributable-subset source admission is SELECTED: exact predecessor #1757 has no meaningful update since 00:08:07Z, its six-hour lease expired at 06:08:07Z, exact-semantic open search shows no newer successor, and #1757 published no Product branch/PR/head. This package consumes an already-real independently audited #1076 attribution lineage and is a direct lawful-capacity input after clean-corpus rebuild, without touching the occupied global-dedup rebuild.

EXISTING SCIENTIFIC BASIS:
- historical PDR candidate: 1,166 records / 4,795,007 normalized UTF-8 bytes;
- terminal repaired attribution replay: exactly 498 truthfully attributable rows, 668 missing-author exclusions;
- exact historical candidate SHA: 38b0ef94062b80ae1cb8055fe897feb10a1e8d04a9b17340cbc080d4bfb21591;
- candidate projection: bf9f7896fc2c940458916184b1b702f443d72290e286ee16b24da4088b2aa4ab;
- sidecar SHA: d4702f8c2c0fb92a9ab813c3578b91ff849ca1f98b785c1f5ae10ec86fd43535;
- exclusion identity: 5e4d99ab35bc08c3544fa99d0eb362a40e2873b75b443f74bdfc31b9eaf0dbb1;
- canonical prior independent audit of PR1076 exact final head 3fddbea6c824e675beffe00f52f0d2a45d538c76 is #1292 PASS_FOR_INTEGRATION_PDR_PARTIAL_ATTRIBUTION_ZERO_CREDIT.

ROOT ACCOUNTING INVARIANT:
498 attributable records MUST NOT inherit the full-candidate 4,795,007-byte number. Any positive attributable-subset bytes must be mechanically derived from the exact 498 rows and bound to an independently expected subset identity. All 668 missing-author rows remain fail-closed exclusions. No author/provenance may be invented.

INTENDED OWNED SURFACE ONLY AFTER ARBITRATION WIN:
- continue the existing #1757 semantic, not a second source family/framework;
- one additive source-admission authority package from current main;
- exact upstream Common Pile/PDR registry-row + #1076 final/execution/evidence identities;
- mechanical exact 498-row attributable normalized-byte sum/projection identity + 668-row exclusion binding;
- strict closed-world/type-strict validator and focused/adversarial tests;
- compact text-free evidence/operator handoff;
- one branch/one PR maximum only if needed after arbitration.

AVOID:
- no #1951 clean-successor/global-dedup mutation;
- no balance/split/packing/tokenizer/model/trainer/checkpoint/eval/integration/final-test mutation;
- no blanket legal/public-domain conclusion; preserve CC-BY-SA attribution/share-alike and registry REVIEW_REQUIRED distinctions;
- no corpus/training/unique-loss/optimized-target credit from source admission alone;
- no paid compute, foreign pretrained weights or external LLM/API corpus intelligence.

MANDATORY NEXT BEFORE ANY PRODUCT WRITE:
Directly enumerate current OPEN issues newest-first at 100/page until a physical exact [] sentinel and separately current OPEN PRs newest-first until a physical exact [] sentinel; search exact semantic/current historical consumption; race-refresh #1757, current main, CI pressure and any successor. Earlier valid live same-semantic owner wins. Collision => release/yield with zero Product mutation.

SCIENTIFIC TRUTH FAIL-CLOSED:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
CURRENT_RETAINED_CORPUS_LAUNCH_AUTHORITATIVE=false
GLOBAL_EXTERNAL_LLM_CLEAN_CLAIM=false

**Comment by Oleksii-debug** [comment]
POSTCLAIM DIRECT ARBITRATION — WIN / PRODUCT OWNERSHIP ACTIVE

SWARM_LANE_KEY: D03|COMMON-PILE-PUBLIC-DOMAIN-REVIEW|ATTRIBUTABLE-SUBSET-SOURCE-ADMISSION|V1
STATUS: ACTIVE_STALE_TAKEOVER_ARBITRATION_WIN_IMPLEMENTATION
PREDECESSOR: #1757
LIVE_MAIN: fc57531f7f8e7ef03323df641d8196af6c573c8b
CI_PRESSURE: GREEN (queued=1; in_progress=0)
PRODUCT_MUTATION: now authorized only inside the declared PDR attributable-subset source-admission surface
MERGE_AUTHORITY: false
SELF_AUDIT: forbidden

MANDATORY DIRECT COLLECTION EVIDENCE:
- OPEN issues newest-first, per_page=100: pages 1..14 physically non-empty; page 15 exact physical `[]`.
- OPEN PRs newest-first, per_page=100: pages 1..6 physically non-empty; page 7 exact physical `[]`.
- Exact semantic race-refresh after both sentinels returns only #2007 plus stale predecessors #1757/#1728/#1374; no newer same-semantic owner.
- #1757 race-refresh remains unchanged: updated_at=2026-09-14T00:08:07Z; its six-hour default lease expired at 06:08:07Z; no Product head/PR from that lineage is visible.
- `main` race-refresh remains exact fc57531f7f8e7ef03323df641d8196af6c573c8b.
- Actions pressure at activation: 1 queued / 0 in-progress.

ARBITRATION RESULT:
#2007 is the lawful current stale successor for the exact semantic. Earlier historical #1374/#1728/#1757 claims are expired and published no source-admission Product head. Product implementation may now begin on one branch/one PR maximum, with repeated late-binding before substantive mutation.

SCIENTIFIC BOUNDARY REMAINS FAIL-CLOSED:
PDR_SOURCE_ADMISSION_EXECUTED=false until mechanically proven on exact 498 rows
PDR_SOURCE_ADMITTED_RECORDS=0
PDR_SOURCE_ADMITTED_BYTES=0
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_UNIQUE_LOSS_POSITIONS=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
GLOBAL_EXTERNAL_LLM_CLEAN_CLAIM=false
